04/app.js

```diff
@@ -5,4 +5,69 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+
+// Sposób drugi
+
+const nav = document.querySelector('nav');
+console.log(nav);
+
+if(nav) {
+    const ul = document.createElement('ul');
+
+    if(ul) {
+        for(let i=1; i<=3; i++) {
+            ul.appendChild(  document.createElement('li') );
+        }
+    }
+
+    nav.appendChild(ul)
+    nav.childNodes[1].remove();
```

Comment:
Wystarczy, że w pętli `for` napiszesz warunek `i<menuItems.length` bez równości i wtedy nie będzie za dużo elementów.

Reply:
Ten warunek nie odnosi się do elementów, ich jest odpowiednia ilość. Usunąłem komentarz, który był bazowo dodany do html, dla przejrzystości w konsoli.